db/zm_create.sql.in

```diff
@@ -344,6 +345,10 @@ CREATE TABLE `Monitors` (
   `Palette` int(10) unsigned NOT NULL default '0',
   `Orientation` enum('0','90','180','270','hori','vert') NOT NULL default '0',
   `Deinterlacing` int(10) unsigned NOT NULL default '0',
+  `SaveJPEGs` TINYINT NOT NULL DEFAULT '3' ,
```

Comment:
Put these into an update file yet or wait until we know what version we are going into?